libs/kerla_utils/bitmap.rs

```diff
@@ -51,6 +50,30 @@ impl<const SZ: usize> BitMap<SZ> {
 
         None
     }
+
+    pub fn assign(&mut self, rhs: [u8; SZ]) {
+        self.0 = rhs;
+    }
+
+    pub fn assign_or(&mut self, rhs: [u8; SZ]) {
+        let mut iter = rhs.into_iter();
+        for byte in &mut self.0 {
+            *byte |= iter.next().unwrap_or(0);
+        }
+    }
+
+    pub fn assign_material_nonimplication(&mut self, rhs: [u8; SZ]) {
+        let mut iter = rhs.into_iter();
+        for byte in &mut self.0 {
+            *byte &= !iter.next().unwrap_or(0);
+        }
```

Comment:
Let's access `rhs` without using iterators and add a doc comment to note that this method would panic if `SZ != rhs.len()`.

```suggestion
        for (i, byte) in self.0.iter_mut().enumerate() {
            *byte &= !rhs[i];
        }
```

Reply:
Done, but I am not sure if I did comment part correctly :)